packages/backend/src/routes/attesterDescription.mjs

```diff
@@ -28,6 +28,12 @@ export default ({ app, db, synchronizer }) => {
     )
 
     const deployer = await getDeployer(BlockExplorer[network], attesterId)
+    console.log('attesterId:', attesterId)
+    console.log('deployer:', deployer)
+    console.log(
+      'signer:',
+      ethers.utils.recoverAddress(hash, signature).toLowerCase()
+    )
```

Comment:
@0xWOLAND the update to `blockExplorer.mjs` resolved the '...not valid JSON' error.
But the signature check is failing. the above console.log gives me this:

> attesterId: 0xaf5c4d939ac218383c8cc0da719888bb20b1c1b3
> deployer: 0x
> signer: 0xb29634a4c3ade98cd0452f02ba051fd974746382

the attesterId passed is the correct address of my attester contract.
deployer does not return an address.
the signer does not match my wallet address (0xb29634A4C3AdE98cd0452f02BA051FD974746382)